package.json

```diff
@@ -122,7 +122,7 @@
   },
   "lint-staged": {
     "*.js": [
-      "lint --fix",
+      "./node_modules/.bin/eslint --fix",
```

Comment:
why not use the `lint-fix` script?

Reply:
lint --fix as well as lint-fix are failing without output - no idea why.
this way it is working ... :man_shrugging: